File: contest/1.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    ios::sync_with_stdio(false);
    cin.tie(0);

    int n;
    cin >> n;

    vector<int> shurs(n, 0);

    int shurn = 1;
    int64_t shursum = 0;
    int64_t res = 0;

    for (int i = 0; i < n; i++){

        shurn = (2*(i+1)-1);
        shursum += pow(shurn, 2);

        res = pow(shurn, 3) - shursum;
        cout << res << ' ';
    }

}

File: contest/1go/1.go
package main

import ( 
	"fmt" 
	"math" 
)

func main() { 
	var n int 
	fmt.Scan(&n)

	shurn := 1
	shursum := 0
	res := 0

	for i := 0; i < n; i++ {
		shurn = 2*(i+1) - 1
		shursum += int(math.Pow(float64(shurn), 2))
		res = int(math.Pow(float64(shurn), 3)) - shursum
		fmt.Print(res, " ")
	}
}